autoload/LanguageClient.vim

```diff
@@ -261,6 +262,154 @@ function! s:GetVar(...) abort
     endif
 endfunction
 
+function! s:ShouldUseFloatWindow() abort
+    return s:FLOAT_WINDOW_AVAILABLE && get(g:, 'LanguageClient_useFloatingHover', 1)
```

Comment:
Can you change it to make use of `s:GetVar`? which would support buffer local variable as well.

Reply:
Yes